sqlite3/lib/src/ffi/constants.dart

```diff
@@ -1,44 +1,247 @@
-// Common result codes, https://www.sqlite.org/rescode.html
-
 import 'dart:ffi';
 
-const SQLITE_OK = 0;
-const SQLITE_ROW = 100;
-const SQLITE_DONE = 101;
+/// Common result codes, https://www.sqlite.org/rescode.html
+/// Result Codes
+///
+/// Many SQLite functions return an integer result code from the set shown
+/// here in order to indicates success or failure.
+///
+/// New error codes may be added in future versions of SQLite.
+///
+/// See also: SQLITE_IOERR_READ | extended result codes,
+/// sqlite3_vtab_on_conflict() SQLITE_ROLLBACK | result codes.
+class SqlErrors {
```

Comment:
Let's call this class `SqlError` which makes it easier to read a specific constant in code.

Reply:
You are right, I've renamed to SqlError